sql/parser/parse_test.go

```diff
@@ -84,6 +84,8 @@ func TestParse(t *testing.T) {
 		{`DROP TABLE a.b`},
 		{`DROP TABLE a, b`},
 		{`DROP TABLE IF EXISTS a`},
+		{`DROP INDEX a.b@c`},
```

Comment:
I'm not sure we have to follow postgres in that behavior. DDL stuff like this is necessarily unique to the database.